### PR TITLE
GPIB-567

### PR DESCRIPTION
**Pull request by xiangzhe0812**
https://dltx.atlassian.net/jira/software/projects/GPIB/boards/6/backlog?selectedIssue=GPIB-567

### CODE REVIEW

File: src/components/addresses/AddressGroupForm.js [hunk 1]
```diff
@@ -3,23 +3,28 @@ import { Formik, Form } from "formik";
...
-const validate = ({ label, address1 }) => {
+const validate = ({ label, address1, percent }) => {
   const errors = {};
   const reqMsg = "This field is required";
   if (!label) errors.label = reqMsg;
   if (!address1) errors.address1 = reqMsg;
+  if (!isNumeric(percent.toString()))
+    errors.percent = "percent must be a number";
+  if (Number(percent) < 0 || Number(percent) > 100)
+    errors.percent = "percent must be between 0 and 100";
```
Comment (Mitchell139) [inline src/components/addresses/AddressGroupForm.js]: Should these have capitals?
```suggestion
    errors.percent = "Percent must be a number";
  if (Number(percent) < 0 || Number(percent) > 100)
    errors.percent = "Percent must be between 0 and 100";
```

**Review by JasminFarhi — DISMISSED** [review]
Nice

**Review by JasminFarhi — DISMISSED** [review]

File: src/components/addresses/AddressGroupForm.js [hunk 2]
```diff
@@ -3,23 +3,28 @@ import { Formik, Form } from "formik";
 import Input from "components/forms/Input";
 import SubmitSpinnerButton from "components/forms/SubmitSpinnerButton";
 import ErrorMessage from "components/ErrorMessage";
+import isNumeric from "validator/lib/isNumeric";
 
 const defaultInitialValues = {
-  percent: 100,
+  percent: "100",
```
Comment (bitcoinbrisbane) [inline src/components/addresses/AddressGroupForm.js]: Why a string?

**Review by bitcoinbrisbane — COMMENTED** [review]
One question, otherwise LGTM

**Review by Mitchell139 — APPROVED** [review]
LGTM

**Review by bitcoinbrisbane — APPROVED** [review]

**Pull request merged**